src/contest.c

```diff
@@ -0,0 +1,5281 @@
+#include "global.h"
+#include "constants/items.h"
+#include "constants/songs.h"
+#include "battle_anim.h"
+#include "contest.h"
+#include "contest_link_80C857C.h"
+#include "data2.h"
+#include "decompress.h"
+#include "ewram.h"
+#include "graphics.h"
+#include "link.h"
+#include "main.h"
+#include "menu.h"
+#include "menu_cursor.h"
+#include "overworld.h"
+#include "palette.h"
+#include "random.h"
+#include "script.h"
+#include "sound.h"
+#include "sprite.h"
+#include "string_util.h"
+#include "task.h"
+#include "text.h"
+#include "tv.h"
+#include "unknown_task.h"
+
+extern bool8 AreMovesContestCombo(u16, u16);
+extern void sub_80C8A38(u8);
+extern void sub_80C8AD0(u8);
+extern void sub_80C8C80(u8);
+
+struct Shared18000
+{
+    /*0x18000*/ u8 unk18000;
+    /*0x18001*/ u8 filler18001[3];
+    /*0x18004*/ u16 unk18004[16][16];
+    /*0x18204*/ u16 unk18204[0x200];
+    /*0x18604*/ u16 unk18604[0x200];
+    /*0x18A04*/ u8 unk18A04[0x800];
+};
+
+struct Shared19204
+{
+    /*0x19204*/ u8 unk19204;
+    /*0x19205*/ u8 unk19205;
+    /*0x19206*/ u8 unk19206[4];
+#if 1
```

Comment:
I'm presuming this is temporary until contest.c is finished?

Reply:
Yes. I can't decide whether it's a bitfield or not.